### PR TITLE
Update README.md to reflect new community contribution guide

### PR DESCRIPTION
**Pull request by jasonmayes**
Updated Pipeline contribution guidelines

### CODE REVIEW

File: pipelines/README.md [hunk 1]
```diff
@@ -1,163 +1,129 @@
...
+   way as the folder, for our example above it would be `cat_on_table_detector.json`.
+   
+   b) **name_of_pipeline.jpg** (*required*) A high resolution JPG image that is 1280x720
+   pixels in size (16:9 ratio) showing a key snapshot of the Visual Blocks pipeline
+   you created in action. This will be shown in search results on the visual blocks
+   community pipelines page.
+   
+   c) **name_of_pipeline.mp4** (*optional*) If you are feeling creative, you can make a
+   short up to 15 seconds max, MP4 video (with no sound) showing the pipeline in action. If
+   you do this, we shall use this once we support it, to enhance your search result with a looped video of
```
Comment (jinjingforever) [inline pipelines/README.md:61]: I see a line break in the rendered result: https://screenshot.googleplex.com/9xTkR9GJEF6gwXx

File: pipelines/README.md [hunk 2]
```diff
@@ -1,163 +1,129 @@
...
-*   [Review Generation and Rating](https://visualblocks.withgoogle.com/#/edit/_?project_json=https:%2F%2Fraw.githubusercontent.com%2Fgoogle%2Fvisualblocks%2Fmain%2Fpipelines%2Fllm%2Freview_generation_and_rating.json)
-    with GPT3.5
+2. **tags** (*required*) - An array of strings containing one or more of the following possible tags:
+     * text
+     * vision
+     * other
 
-    [![Review Generation and Rating](llm/review_generation_and_rating.jpg)](https://visualblocks.withgoogle.com/#/edit/_?project_json=https:%2F%2Fraw.githubusercontent.com%2Fgoogle%2Fvisualblocks%2Fmain%2Fpipelines%2Fllm%2Freview_generation_and_rating.json)
+   For example, if your pipeline was solely around vision, you would just enter `[“vision"]` for this field.
+   If your pipeline covered multiple areas, for example a speech to text pipeline's categories might be `["audio”, “text"]` as the string array for this field.
```
Comment (jinjingforever) [inline pipelines/README.md:96]: Since we don't have "audio" in the tag list, maybe we could update this example to use "vision" and "text"?

**Review by jinjingforever — APPROVED** [review]
Thank you Jason! Some minor comments below

**Review by jasonmayes — COMMENTED** [review]
Fixed break and tag usage example

**Review by jinjingforever — APPROVED** [review]

**Pull request merged**